### ISSUE TITLE
Omen Globe support

### ISSUE THREAD

**Issue by stupxd** [MathIsFun0/Immolate#14]
Support using immolate to find Spectral cards from Arcana packs

**Comment by dev-asya** [comment]
Do you mean to find specific cards, or just the presence of Spectral Packs?

**Comment by stupxd** [comment]
> Do you mean to find specific cards, or just the presence of Spectral Packs
> 
> EDIT: I can't read! my bad

Yeah, there's a function to generate a spectral pack cards, so updating that function to include all spectral cards that are generated (including black hole, I guess)
